src/components/SozialMarie/index.jsx

```diff
@@ -0,0 +1,164 @@
+import useTimer from 'hooks/useTimer';
+import { Alert, Box, Divider, Snackbar, Stack } from '@mui/material';
+import { useLocalStorage } from 'hooks';
+import { useEffect, useState, useCallback } from 'react';
+import { t } from 'i18next';
+import VotingButton from './VotingButton';
+import AlertCountDown from './AlertCountDown';
+import SozialMarieLink from './SozialMarieLink';
+import AlertFooterContent from './AlertFooterContent';
+import AlertContentHeader from './AlertHeaderContent';
+import { startDate, endDate } from './date-range';
+
+const DELAY_TO_HIDE_ALERT = 5000;
+const DELAY_TO_HIDE_TRIGGER = 6000;
+
+if (DELAY_TO_HIDE_ALERT > DELAY_TO_HIDE_TRIGGER) {
+  throw new Error('DELAY_TO_HIDE_ALERT should be less than DELAY_TO_HIDE_TRIGGER');
+}
+
+const SOZIAL_MARIE_LINK = 'https://www.sozialmarie.org/sl';
+
+const SozialMarie = function SozialMarie() {
+  const currentDate = new Date();
+  const countDownDate = currentDate < startDate ? startDate : endDate;
+  const isVoting = currentDate >= startDate && currentDate < endDate;
+  const isBefore = currentDate < startDate;
+  const isAfter = currentDate >= endDate;
+  const roundedInitialTime = Math.floor((countDownDate - currentDate) / 1000) * 1000;
+  const [timeLeft, setTimeLeft] = useTimer(roundedInitialTime);
+
+  const [show, updateShow] = useLocalStorage('showSozialMarie', 'first');
```

Comment:
Popup is shown even if user ticks the checkbox to not show it again and the value "no-show" is persisted into the local storage.

Reply:
It looks like I fixed it.

> Alternatively the text should be "Don't show this again until the voting starts", so that the users are reminded again when they can vote and then they can silence the popup for good.

Before voting starts the value is "remind-me" and during voting "no-show".